### ISSUE TITLE
PlexRipper setup process for Synology

### ISSUE THREAD

**Issue by JasonLandbridge** [Reaparr/Docs#9]
@talormanda, you mentioned here: https://github.com/PlexRipper/PlexRipper/issues/168 that you might be able to make a Synology setup guide. It would still be greatly appreciated if you have the time for it!

**Comment by talormanda** [comment]
> 

yeah i havent forgot lol. where would i send it?

**Comment by JasonLandbridge** [comment]
Thanks!

You have two options:

You could go to the [Synology page](https://www.plexripper.rocks/guides/installation/synology) and click on "[Edit this page on GitHub](https://github.com/PlexRipper/Docs/edit/master/src/content/guides/3.installation/6.synology.md)". Then you can fill in the page by using [Markdown](https://www.youtube.com/watch?v=f49LJV1i-_w). You can even copy and paste images into the text which will then auto upload them as well. 

Second option is, you can make a Microsoft Office Word document or a PDF, but then the images and styling will need to be manually copied over into Markdown anyway.

Let me know if you have further questions!

**Comment by patrickronk** [comment]
Did a setup guide for Synology ever get produced? I'm painfully oblivious about how most of this stuff works and could use some guidance. Unfortunately, the Synology page is still blank.

**Comment by JasonLandbridge** [comment]
Nope, @talormanda never made it despite promising it multiple times. You can ask CHATGPT to guide you through it though

**Comment by JasonLandbridge** [comment]
Added an AI guide for now: https://www.reaparr.rocks/guides/installation/synology
